Week1/homework/app.js

```diff
@@ -1,11 +1,171 @@
+      author: 'S. Nursi',
+    },
+    {
+      id: 'safahat',
+      title: 'Safahat',
+      language: 'Turkish',
+      author: 'M.A. Ersoy',
+    },
+    {
+      id: 'towards_the_lost_paradise',
+      title: 'Towards The Lost Paradise',
+      language: 'English',
+      author: 'F. Gulen',
+    },
+    {
+      id: 'sapiens',
+      title: 'Sapiens',
+      language: 'English',
+      author: 'Yufal Noah Harari',
+    },
+    {
+      id: 'facade',
+      title: 'Facade',
+      language: 'English',
+      author: 'Esther Verhoef',
+    },
+    {
+      id: 'love_and_tolerance',
+      title: 'LOve And Tolarance',
+      language: 'English',
+      author: 'F. Gulen',
+    },
+    {
+      id: 'from_seed_to_cedar',
+      title: 'From Seed To Cedar',
+      language: 'Turkish',
+      author: 'F.Gulen',
+    },
   ];
 
-  // Replace with your own code
-  console.log(bookTitles);
+  const imageLink = {
+    harry_potter_chamber_secrets: {
+      link: './img/harry_potter_chamber_of_the_secrets.jpg',
+      alt: 'harry potter',
+    },
+    sufisim: {
+      link: './img/sufism.jpg',
+      alt: 'sufism',
+    },
+    the_statue_of_our_souls: {
+      link: './img/the_statue_of_our_souls.jpg',
+      alt: 'the statue of our souls',
+    },
+    the_words: {
+      link: './img/the_words.jpeg',
+      alt: 'harry potter',
+    },
+    safahat: {
+      link: './img/safahat.jpeg',
+      alt: 'safahat',
+    },
+    towards_the_lost_paradise: {
+      link: './img/towards_the_lost_paradise.jpg',
+      alt: 'towards the lost paradise',
+    },
+    sapiens: {
+      link: './img/sapiens.jpeg',
+      alt: 'sapiens',
+    },
+    facade: {
+      link: './img/facade.jpg',
+      alt: 'facade',
+    },
+    love_and_tolerance: {
+      link: './img/love_tolerance.jpg',
+      alt: 'love and tolerance',
+    },
+    from_seed_to_cedar: {
+      link: './img/from_seed_to_cedar.jpg',
+      alt: 'from seed to cedar',
+    },
+  };
+
+  function getBookInfo(obj, img) {
+    const bookList = document.createElement('ul');
+    rootDiv.appendChild(bookList);
+
+    for (let i = 0; i < obj.length; i++) {
+      const bookTitle = document.createElement('li');
+      bookTitle.setAttribute('class', 'li-bookTitle');
+
+      bookList.appendChild(bookTitle);
+      const bookName = document.createElement('h2');
+      bookTitle.appendChild(bookName);
+      bookName.innerHTML = obj[i].title;
+      bookName.setAttribute('class', 'h2-bookName');
+
+      const author = document.createElement('p');
+      bookTitle.appendChild(author);
+      author.innerHTML = obj[i].author;
+      author.setAttribute('class', 'p-author');
+
+      const language = document.createElement('p');
+      bookTitle.appendChild(language);
+      language.innerHTML = obj[i].language;
+      language.setAttribute('class', 'p-language');
+
+      const coverImage = document.createElement('img');
+      bookTitle.appendChild(coverImage);
+      const imgTarget = obj[i].id;
```

Comment:
You should create a separate loop to add the cover images like it is mentioned in the instructions.

Reply:
I am so sorry.
I have already implemented the changes more than a week ago. I forget to inform you about it, terribly sorry for that.
They are ready for your review and approve, please.